forge-gui/res/cardsfolder/upcoming/mirkwood_trapper.txt

```diff
@@ -0,0 +1,12 @@
+Name:Mirkwood Trapper
+ManaCost:1 G U
+Types:Creature Elf Scout
+PT:1/4
+T:Mode$ AttackersDeclared | ValidCard$ Creature | Attacked$ You | TriggerZones$ Battlefield | Execute$ TrigPump | TriggerDescription$ Whenever a player attacks you, target attacking creature gets -2/-0 until end of turn.
+SVar:TrigPump:DB$ Pump | ValidTgts$ Creature.attacking | NumAtt$ -2 | IsCurse$ True
+T:Mode$ AttackersDeclared | ValidAttackers$ Creature.YouCtrl | AttackedTarget$ Player.Other | Execute$ TrigChoose | TriggerZones$ Battlefield | TriggerDescription$ Whenever a player attacks, if they aren't attacking you, that player chooses an attacking creature. It gets +2/+0 until end of turn.
```

Comment:
the Valid doesn't make sense
+ you need to write it as condition instead or an opponent could get both triggers at the same time